src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10690,6 +10786,27 @@ definitions:
         $ref: '#/definitions/PermissionsInfo'
       definition:
         $ref: '#/definitions/Definition'
+  calculateSizeResponse:
+    type: object
+    properties:
+      id:
+        type: string
+        description: Provides the id for which POST/calculateSize has been initiated.
+      size:
+        type: string
+        description: Provides a folder size in bytes.
+      calculatedAtTime:
+        type: string
+        description: Provides the time when the calculating folder size will be completed.
+      status:
+        type: string
```

Comment:
I think we should have "number of files" field in the response. This was earlier part of confluence as well and is in sync with the requirement from Product Manager.

Reply:
Added the same.
Done